packages/test/src/test-worker-tuner.ts

```diff
@@ -0,0 +1,154 @@
+import test from 'ava';
+import { v4 as uuid4 } from 'uuid';
+import { Client } from '@temporalio/client';
+import { ResourceBasedTunerOptions } from '@temporalio/core-bridge';
+import { defaultOptions } from './mock-native-worker';
+import { RUN_INTEGRATION_TESTS, Worker } from './helpers';
+import { successString } from './workflows';
+
+if (RUN_INTEGRATION_TESTS) {
```

Comment:
No worry for this file, but in the future, please uses `test-integration-workflows.ts` as a base for integration tests. We plan to completely remove the older pattern.